apps/avifenc.c

```diff
@@ -388,12 +393,21 @@ int main(int argc, char * argv[])
     avifRWData exifOverride = AVIF_DATA_EMPTY;
     avifRWData xmpOverride = AVIF_DATA_EMPTY;
     avifRWData iccOverride = AVIF_DATA_EMPTY;
-    int duration = 1;  // in timescales, stored per-inputFile (see avifInputFile)
-    int timescale = 1; // 1 fps by default
     int keyframeInterval = 0;
     avifBool cicpExplicitlySet = AVIF_FALSE;
     avifBool premultiplyAlpha = AVIF_FALSE;
 
+    // The timescale specifies how many units of time are in a second, and the duration is how many
+    // of those timescales this frame should be displayed for. For example, if they match, it signals
```

Comment:
I suggest changing "timescales" to "time units" or "units of time". The wording used in ISO BMFF is "in the indicated timescale".

Alternatively, we can make a pass through the source tree later to correct this misuse of terminology.

Reply:
I think this is resolved as I rewrote the comment entirely when I moved it to above the new `avifAppSourceTiming` declaration. Let me know if it needs more tweaks.